src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java

```diff
@@ -25,6 +40,10 @@
     static final String currentUserDirectory = "CurrentUser";

```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocVariableCheck
**Severity**: ERROR
**File**: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java L40

Missing a Javadoc comment.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-1319891517>*